pkg/crc/machine/kubeconfig.go

```diff
@@ -10,7 +10,7 @@ import (
 	"net/url"
 	"os"
 	"path/filepath"
-	"strconv"
+	`strconv`
```

Comment:
unrelated change probably by the ide?

Reply:
@vyasgun can you fix it and repush it?

Reply:
@vyasgun can you handle this ASAP, otherwise it will not be possible to merge in time for the next release

Reply:
did linting fail? otherwise we can merge regardless and fix this ASAP if not done today.

Reply:
This PR has " Maintainers are allowed to edit this pull request.", I've fixed this.

Reply:
thanks @cfergeau! Sorry I was AFK